files/scripts/syncd.sh

```diff
@@ -98,11 +98,18 @@ start() {
         BOOT_TYPE=`getBootType`
         if [[ x"$WARM_BOOT" == x"true" || x"$BOOT_TYPE" == x"fast" ]]; then
             export FAST_BOOT=1
+        else
+            /bin/systemctl stop pmon
+            /usr/bin/hw-management.sh chipdown
```

Comment:
where is the chipup?

Reply:
@liat-grozovik done internally by hw-mgmt to speed up syncd start.